ui/articles-aem-renderer/src/main/java/org/cru/godtools/articles/aem/service/support/ArticleParser.java

```diff
@@ -136,7 +148,13 @@ private static Article parseArticleObject(JSONObject articleObject) {
     @NonNull
     private static List<Attachment> getAttachmentsFromRootObject(JSONObject articleRootObject, String articleKey) {
```

Comment:
instead of `String articleKey`, I would pass in the current `baseUri` that you have been building.